dipy/reconst/dki_micro.py

```diff
@@ -320,7 +324,8 @@ class KurtosisMicrostructureModel(DiffusionKurtosisModel):
     """Class for the Diffusion Kurtosis Microstructural Model"""
 
     def __init__(self, gtab, fit_method="WLS", *args, **kwargs):
-        """Initialize a KurtosisMicrostrutureModel class instance [1]_.
+        """Initialize a KurtosisMicrostrutureModel class instance
+        :footcite:p:`Fieremans2011`_.
```

Comment:
This is probably an error
```suggestion
        :footcite:p:`Fieremans2011`.
```

Reply:
:+1: Fixed in 8412ffb.